src/github_runner_manager/openstack_cloud/openstack_runner_manager.py

```diff
@@ -636,7 +639,15 @@ def _wait_runner_startup(self, instance: OpenstackInstance) -> None:
                 "not completed"
             ) from err
 
-        result: invoke.runners.Result = ssh_conn.run("ps aux", warn=True)
```

Comment:
Note:
I cannot re-use the `_run_health_check` method here because it also checks whether for runner.Worker, not just runner.Listener.